troposphere/static/js/components/Header.jsx

```diff
@@ -114,19 +114,11 @@ let LogoutLink = React.createClass({
     },
 
     render: function() {
-        let statusPageEl;
         let username = this.props.username;
 
         if (!username && window.show_public_site) {
             username = "AnonymousUser"
         }
-        if (globals.STATUS_PAGE_LINK) {
-            statusPageEl = (
-                <li>
-                    <a href={globals.STATUS_PAGE_LINK} target="_blank">Status</a>
```

Comment:
So the href is coming from globals.STATUS_PAGE_LINK, this implies that the href is still out there.